app/src/components/formdemo/FormDemo.tsx

```diff
@@ -646,6 +650,81 @@ export default function FormDemo() {
 		),
 	});
 
+	demoSections.push({
+		title: 'CopyButton',
+		content: (
+			<Fragment>
+				<p>A copy button</p>
+				<h3>Example</h3>
+				<pre>
+					<code>{`<CopyButton 
+	copyText='A text example' 
```

Comment:
Make sure you've installed the Prettier extension for VS Code and you run the formatter on these files. The indentation isn't correct. It's just a display issue, but it makes the code more difficult to read.